File: python/tr20.py

usuario_correcto = "admin"
contrasena_correcta = "1234"
usuario = input("Ingrese usuario: ")
contrasena = input("Ingrese contraseña: ")

if usuario == usuario_correcto and contrasena == contrasena_correcta:
    print("Acceso permitido")
else:
    print("Usuario o contraseña incorrectos")

File: java script/tr20.js
let usuarioGuardado = "admin";
let claveGuardada = "1234";
let usuario = prompt("Ingrese el usuario");
let clave = prompt("Ingrese la contraseña");
if (usuario == usuarioGuardado && clave == claveGuardada) {
    alert("Acceso permitido");
} else {
    alert("Usuario o contraseña incorrectos");
}